src/twisted/internet/base.py

```diff
@@ -893,7 +893,7 @@ def argChecker(arg):
                     return None
                 warnings.warn(
                     "Argument strings and environment keys/values passed to "
-                    "reactor.spawnProcess should be str, not unicode.",
+                    "reactor.spawnProcess should be bytes, not unicode.",
```

Comment:
if we go with `unicode` instead of native `string/unicode` generally, then it's fine here, too, and anyway this is obviously an improvement.